.golangci.example.yml

```diff
@@ -579,9 +580,8 @@ linters-settings:
     # You can specify idiomatic endings for interface
     - (or|er)$
 
-    # Reject patterns
-    reject:
-    - github.com\/user\/package\/v4\.Type
+    # reject:
+    # - github.com\/user\/package\/v4\.Type
```

Comment:
can you restore the previous section? and uncomment the `keyword` section.

Reply:
done